GiTinder.Tests/Models/SettingsTest.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Xunit;
+
+namespace GiTinder.Tests.Models
+{
+    public class SettingsTest
+    {
+
+        [Fact]
+        public void CanCreateSettingsWithNullUserName()
+        {
+            var settings = SettingsFactory.CreateSettings();
+            Assert.True(settings.UserName == null);
+        }
+
+        [Fact]
+        public void CanSetUserNameWithString()
+        {
+            var settings = SettingsFactory.CreateSettings();
+            settings.UserName = "Pablos";
+            Assert.Equal("Pablos", settings.UserName);
+        }
```

Comment:
@FilipTeply these are the two tests I told Jonathan as well that are unnecessary, since they're testing the language not your code. Remove these

Reply:
I removed the two tests.